cot/Cargo.toml

```diff
@@ -72,10 +72,13 @@ url = { workspace = true, features = ["serde"] }
 
 [dev-dependencies]
 async-stream.workspace = true
+criterion = { workspace = true, features = ["async_tokio"] }
 fake.workspace = true
 fantoccini.workspace = true
 futures.workspace = true
 mockall.workspace = true
+reqwest = { version = "0.11", features = ["json"] }
```

Comment:
Why are we using an almost ancient version of `reqwest`? 0.12 is over 1 year old.